com.microsoft.mrtk.input/Interactors/InteractorVisuals/BaseReticleVisual.cs

```diff
@@ -16,48 +17,86 @@ namespace Microsoft.MixedReality.Toolkit.Input
     [DisallowMultipleComponent]
     public class BaseReticleVisual : MonoBehaviour, IXRCustomReticleProvider
     {
+        [SerializeField]
+        [Tooltip("The root of the reticle visuals")]
+        private Transform reticleRoot;
+
+        /// <summary>
+        /// The root of the reticle visuals. 
+        /// </summary>
+        /// <remarks>
+        /// This transform hold both the base and custom reticle.
+        /// </remarks>
+        protected Transform ReticleRoot => reticleRoot;
+
         [SerializeField]
         [Tooltip("The reticle model to use when the interactable doesn't specify a custom one.")]
         private GameObject baseReticle;
 
+        /// <summary>
+        /// The reticle model to use when the interactable doesn't specify a custom one.
+        /// </summary>
+        protected GameObject BaseReticle => baseReticle;
+
         /// <summary>
         /// Staging area for custom reticles that interactors can attach to show unique visuals.
         /// </summary>
-        protected GameObject customReticle;
+        protected GameObject CustomReticle
+        {
+            get;
+            private set;
+        }
 
         /// <summary>
         /// Is there a custom reticle currently attached to this interactor?
         /// </summary>
-        protected bool customReticleAttached;
+        protected bool CustomReticleAttached
+        {
+            get;
+            private set;
+        }
 
         /// <summary>
         /// The current reticle that the interactor is using.
         /// </summary>
-        public GameObject Reticle => customReticleAttached ? customReticle : baseReticle;
+        public GameObject Reticle => CustomReticleAttached ? CustomReticle : baseReticle;
 
-        private IVariableReticle variableReticle;
+        private IReticleVisual visual;
 
         /// <summary>
-        /// Cached variable reticle reference.
+        /// Cached reference to the <see cref="IReticleVisual"/> component on <see cref="Reticle"/>.
         /// </summary>
-        protected IVariableReticle VariableReticle
+        protected IReticleVisual Visual
```

Comment:
Name: ReticleVisual?

Reply:
The class is already has the word "Reticle" in it...So I felt that adding reticle would be redundant.   But I was also not happy with `Visual` since the class is called `BaseReticleVisaul`...

How strongly do you feel about this?  Or any other ideas?